dialog/file.go

```diff
@@ -559,21 +559,31 @@ func (f *FileDialog) SetFileName(fileName string) {
 }
 
 // NewFileOpen creates a file dialog allowing the user to choose a file to open.
+// The callback function will run when the dialog closes.  The URI will be nil
+// when the user cancels or when nothing is selected.
+//
 // The dialog will appear over the window specified when Show() is called.
 func NewFileOpen(callback func(fyne.URIReadCloser, error), parent fyne.Window) *FileDialog {
 	dialog := &FileDialog{callback: callback, parent: parent}
 	return dialog
 }
 
-// NewFileSave creates a file dialog allowing the user to choose a file to save to (new or overwrite).
-// If the user chooses an existing file they will be asked if they are sure.
+// NewFileSave creates a file dialog allowing the user to choose a file to save
+// to (new or overwrite).  If the user chooses an existing file they will be
+// asked if they are sure.  The callback function will run when the dialog
+// closes.  The URI will be nil when the user cancels or when nothing is
+// selected.
+//
 // The dialog will appear over the window specified when Show() is called.
 func NewFileSave(callback func(fyne.URIWriteCloser, error), parent fyne.Window) *FileDialog {
 	dialog := &FileDialog{callback: callback, parent: parent, save: true}
 	return dialog
 }
 
-// ShowFileOpen creates and shows a file dialog allowing the user to choose a file to open.
+// ShowFileOpen creates and shows a file dialog allowing the user to choose a
+// file to open. The callback function will run when the dialog closes.  The URI
```

Comment:
```suggestion
// file to open. The callback function will run when the dialog closes. The URI
```

It's grammatically incorrect with double spaces. There are a few more places where this occurs but I just made one comment about it.

Reply:
There is actually a school of thinking that says double space is grammatically correct. However I agree that we have settled on single spacing.

Reply:
I did not know that. It is very, very forbidden in Swedish at least 😅